File: Examples/php/text_processing.php
<?php
$text = "The quick brown fox jumps over the lazy dog";
$words = explode(" ", $text);
$lengths = [];
foreach ($words as $word) {
    $key = strtolower($word);
    $lengths[$key] = strlen($word);
}
ksort($lengths);
foreach ($lengths as $word => $length) echo "$word($length) ";
echo "\n";
$longest = "";
foreach ($words as $word) if (strlen($word) > strlen($longest)) $longest = $word;
echo $longest, " ", str_word_count($text), " ", ucwords(strtolower($text)), "\n";
echo wordwrap($text, 15, "|"), "\n";
$vowels = 0;
foreach (str_split(strtolower($text)) as $character) {
    if (strpos("aeiou", $character) !== false) $vowels++;
}
echo "vowels=$vowels\n";
echo str_replace(" ", "_", $text), "\n";
echo strtr_like($text), "\n";
function strtr_like($text) { return strrev(strtoupper(substr($text, 0, 9))); }

File: Examples/swift/text_processing.swift
import Foundation
let text = "the quick brown fox jumps over the lazy dog"
let words = text.split(separator: " ").map { String($0) }
print(words.count, words.first ?? "", words.last ?? "")
var counts: [String: Int] = [:]
for word in words {
    counts[word] = (counts[word] ?? 0) + 1
}
print(counts["the"] ?? 0)
let longest = words.reduce("") { $1.count > $0.count ? $1 : $0 }
print(longest, longest.count)
print(text.uppercased().prefix(9))
print(text.replacingOccurrences(of: " ", with: "-"))
print(text.hasPrefix("the"), text.contains("fox"), text.count)
var reversedWords: [String] = []
for word in words { reversedWords.append(String(word.reversed())) }
print(reversedWords.joined(separator: " "))
let vowels = text.filter { "aeiou".contains($0) }
print(vowels.count)
